src/Service.Tests/Unittests/SerializationDeserializationTests.cs

```diff
@@ -305,6 +369,45 @@ private void InitializeObjects()
             _databaseStoredProcedure.StoredProcedureDefinition.Parameters.Add("Id", _parameterDefinition);
         }
 
+        /// <summary>
+        /// During serialization we add TypeName to the DatabaseObject based on its type.
+        /// If the code refactor then this change will catch that as GraphQL workload would be impacted.
+        /// We will have artifacts with TypeName with old project directory/ location of this class, hence during deserialization if object is not in
+        /// that location it will fail.
+        /// [Note] : is this test fails notify the graphql workload team.
+        /// </summary>
+        private void TestTypeNameChanges(DatabaseObject databaseobject, string objectName)
```

Comment:
please provide an example or two of what this test validates and what it is searching for in serialized object json string

Reply:
added